src/server/scripts/Northrend/VioletHold/violet_hold.cpp

```diff
@@ -472,12 +472,13 @@ class npc_sinclari_vh : public CreatureScript
                             task.Repeat(Seconds(5));
                             break;
                         case 8:
-                            me->SetVisible(false);
+                            _instance->SetData(DATA_MAIN_EVENT_STATE, IN_PROGRESS);
+                            // [1] GUID: Full: 0xF1300077C202E6DD Type: Creature Entry: 30658 Low: 190173
```

Comment:
well thats kind of a pointless comment, it brings no info to the script :D it doesn't even have the rest of packet data (most likely was part of UPDATE_OBJECT, in which case I would expect to see some field changes)